spec/features/editing_a_manual_section_spec.rb

```diff
@@ -31,7 +31,6 @@
 
       expect(page.status_code).to eq(200)
 
-      expect(page).to have_field('Title')
       fill_in "Title", with: "My updated first section"
```

Comment:
It would probably be better if you could make this change in the [original commit](https://github.com/alphagov/specialist-publisher/pull/689/commits/1f0346f0c7f627321d4b6a6d8dee7e088d841b83) where this was introduced to avoid confusion.

You can also remove the similar lines that check for 'Summary' and 'Body' fields.